src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -1976,6 +2215,43 @@ class A
                   public delegate int D();
                 }
                 """);
+            TestNormalizeDeclaration(
+                "class A{public string Prop{get{}}public delegate int D();}", """
+                class A
+                {
+                  public string Prop
+                  {
+                    get
+                    {
+                    }
+                  }
+
+                  public delegate int D();
+                }
+                """);
+            TestNormalizeDeclaration(
+                """class A{public string Prop{get{}}="xyz";public delegate int D();}""", """
+                class A
+                {
+                  public string Prop
+                  {
+                    get
+                    {
+                    }
```

Comment:
Consider testing things like:

```csharp
public string Prop
{
    get;
    set;
}
```

```csharp
public string Prop
{
    get => string.Empty;
    set _ = value;
}
```

Reply:
the single line form seems fine to me.  i assume you don't get it if the accessors have bodies?